comid/measurement_test.go

```diff
@@ -217,29 +328,28 @@ func TestUIntMkey_FromJSON_ok(t *testing.T) {
 	}
 }
 
-func TestUIntMkey_FromCBOR_ok(t *testing.T) {
+func TestUIntMkey_FromJSON_notok(t *testing.T) {
 	tvs := []struct {
 		input    []byte
-		expected uint64
+		expected string
 	}{
 		{
-			input:    []byte{0x18, 0x64},
-			expected: goldenMKey,
+			input:    []byte("{\"type\":\"uint\",\"value\":\"abcdefg\"}"),
```

Comment:
you can use back-ticks to encode a raw string (and thus avoid escaping).

Reply:
Agree, verified and done now! thanks!